tools/autodeployment/server.js

```diff
@@ -3,7 +3,7 @@ const http = require('http');
 const createHandler = require('github-webhook-handler');
 const shell = require('shelljs');
 
-const { SECRET, REPO_NAME, DEPLOY_PORT } = process.env;
+const { SECRET, REPO_NAME, DEPLOY_PORT, DEPLOY_TYPE } = process.env;
```

Comment:
Please update env.staging and env.production to add this variable with the correct value.

Reply:
Added two new files `env.production` + `env.staging`